admin/base/settings/defaults.py

```diff
@@ -20,6 +20,15 @@
 DEBUG = osf_settings.DEBUG_MODE
 DEBUG_PROPAGATE_EXCEPTIONS = True
 
+# secure mode for using https in local development
+SECURE = osf_settings.SECURE_MODE
+LOCAL = osf_settings.LOCAL_MODE
```

Comment:
`LOCAL` will be removed.
I prefer to keep admin and osf sharing the same settings since they share other services. `SECURE` can be removed, `SESSION_COOKIE_SECURE = osf_settings.SECURE_MODE`.